lib/context/color.ts

```diff
@@ -281,6 +281,7 @@ export class ColorContextConsumer extends ColorContextController implements Reac
         } else {
           this.host.setAttribute(this.attribute, next);
         }
+        this.host.requestUpdate();
```

Comment:
Can you move this to lines 80 and 73?

Reply:
Ok, gotcha as we don't need to request the update on an attribute change ✅